internal/gen/connect/ping/v1/pingv1connect/ping.connect.go

```diff
@@ -12,10 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The canonical location for this file is
+// https://github.com/bufbuild/connect-go/blob/main/internal/proto/connect/ping/v1/ping.proto.
+
 // Code generated by protoc-gen-connect-go. DO NOT EDIT.
 //
 // Source: connect/ping/v1/ping.proto
 
+// The connect.ping.v1 package contains an echo service designed to test the
+// connect-go implementation.
```

Comment:
It looks like the go code adds a trailing space here before the package statement.

Reply:
Yeah. I'm not sure why - the commits over there don't clarify. It looks like this is perhaps an artifact of reusing the same function for the comments on the protobuf package and the comments on the protobuf syntax. For us, it seems nice to produce Go code with package docs.